assets/overlays/aws-efs/generated/standalone/lease_leader_election_binding.yaml

```diff
@@ -0,0 +1,20 @@
+# Generated file. Do not edit. Update using "make update".
+#
+# Loaded from base/rbac/lease_leader_election_binding.yaml
+#
+#
+# Grant controller access to leases
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: aws-efs-csi-driver-lease-leader-election
+  namespace: openshift-cluster-csi-drivers
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: aws-efs-csi-driver-lease-leader-election
+subjects:
+- kind: ServiceAccount
+  name: aws-efs-csi-driver-controller-sa
+  namespace: openshift-cluster-csi-drivers
```

Comment:
I think you want either this yaml to use `${NAMESPACE}` like `controller_sa.yaml` does, or have them both hardcode `openshift-cluster-csi-drivers`.

Same goes for `main_provisioner_binding.yaml`. I could see this breaking things if I try to install to a different namespace, the ServiceAccount gets created in my custom namespace, and the bindings still reference `openshift-cluster-csi-drivers`.

Reply:
I've added a patch for this to keep the base unchanged: https://github.com/openshift/csi-operator/pull/237/commits/88c01c25b55540e0e3b1bc41961d41a3c33cbde5

Also, we have the namespaces hardcoded for samba operator too. Since it's also OLM based it could be installed into custom namespace and so might need the same fix. @mpatlasov WDYT?

Reply:
@RomanBednar , why [88c01c2](https://github.com/openshift/csi-operator/commit/88c01c25b55540e0e3b1bc41961d41a3c33cbde5) doesn't change `overlays/aws-efs/generated/standalone/node_privileged_binding.yaml` 
? Is this an overlook?